buildkite/src/Constants/DebianChannel.dhall

```diff
@@ -21,6 +22,7 @@ let lowerName = \(channel : Channel) ->
     , Nightly = "nightly"
     , Itn = "itn"
     , Umt = "umt"
+    , UmtMainnet = "umt-mainnet"
```

Comment:
Will the hyphen cause any issues with the promotion pipeline? If not this looks good to me!

Reply:
I think dashes and camelCase are supported